hands-on/make.gcc-nv.inc

```diff
@@ -1,3 +1,7 @@
 CXX=g++
 CXX_FLAGS=-g -O3 -fopenmp
 CXX_OFFLOAD_FLAGS=-foffload=nvptx-none
+
+FXX=gfortran
```

Comment:
CXX means C++ compiler. Let's use FC for fortran compiler.

Reply:
fortran compiler has been updated to FC in makefile